### DISCUSSION TITLE
UHD8000

### DISCUSSION THREAD

**Discussion by flaviolanteri** [wizmo2/zidoo-player#36] [General]
Hi,
I use the integration on my UHD8000 and everything works like a charm. However, some commands on the UHD8000 are missing, among which the two most important are:
- Zoom that allows you to vary the zoom of the image
- Resolution that allows you to vary the resolution output from the player 
Is it possible to integrate these commands?
Thank's
Flavio.

**Comment by wizmo2** [comment]
Check the remote button commands.  Dont have the newer units to test.

**Comment by wizmo2** [comment]
Flavio,  I've released a pre-release 2.0.2 which includes a Set Zoom Mode service.

The resolution is a little more complicated as it uses system calls.  I try to avoid these calls within the HA as they could have unwanted affects.  You can use the remote command for this.

P.S. Its also worth considering using the auto resolution option in the zidoo advanced menus.  Its typically provides much better video quality than trying to have the player up/downscale.

**Comment by flaviolanteri** [comment]
Thank's. I will test.
